auto_submit/lib/model/big_query_revert_request_record.dart

```diff
@@ -0,0 +1,98 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+class RevertRequestRecord {
```

Comment:
We could clean this up by extending `PullRequestRecord` since it inherits most of the metadata

Reply:
Let me see what I can do.

Reply:
Done.